parsers/ES_IB.py

```diff
@@ -0,0 +1,99 @@
+# The arrow library is used to handle datetimes
+from arrow import get, utcnow
+# The request library is used to fetch content through HTTP
+from requests import Session
+from reescraper import BalearicIslands
+
+
+def fetch_consumption(country_code='ES-IB', session=None):
+    ses = session or session()
+    response = BalearicIslands(ses).get()
+
+    if not response:
+        datetime = utcnow().datetime
+    else:
+        datetime = get(response.timestamp).datetime
+
+    data = {
+        'countryCode': country_code,
+        'datetime': datetime,
+        'source': 'demanda.ree.es'
+    }
+
+    if response:
+        data['consumption'] = response.demand
+
+    return data
+
+
+def fetch_production(country_code='ES-IB', session=None):
+
+    ses = session or session()
+    response = BalearicIslands(ses).get()
+
+    if not response:
+        datetime = utcnow().datetime
+    else:
+        datetime = get(response.timestamp).datetime
+
+    data = {
+        'countryCode': country_code,
+        'datetime': datetime,
+        'production': {
+          'biomass': 0.0,
+          'nuclear': 0.0,
+          'geothermal': 0.0
+        },
+        'storage': {},
+        'source': 'demanda.ree.es',
+    }
+
+    if response:
+        data['production']['coal'] = response.carbon
+        data['production']['gas'] = response.gas + response.combined
+        data['production']['solar'] = response.solar
+        data['production']['oil'] = response.vapor + response.diesel
+        data['production']['wind'] = response.wind
+
+        hidro = response.hydraulic
```

Comment:
Is it possible de separate pumped hydro from river?
If this is only pumped hydro, then production from pumped hydro should be counted as negative storage and not positive production (like drawing power from a battery)